syntax = "proto3";

message File {
    string user_id = 1;
    string file_path = 2;
}

message FileStructure {
}

service FileStructureService {
    rpc ExtractStructure(stream File) returns (stream FileStructure);
}